src/test/testSchemaGenerator.ts

```diff
@@ -2006,3 +2008,211 @@ describe('chainResolvers', () => {
     ).to.equals('tony');
   });
 });
+
+
+
+describe('attachDirectives', () => {
+  const testSchemaWithDirectives = `
+    directive @upper on QUERY | FIELD
```

Comment:
I think this should just be `on FIELD_DEFINITION`.

Reply:
Edited